core/domain/collection_services.py

```diff
@@ -188,8 +234,20 @@ def get_collection_summary_by_id(collection_id):
 
 def get_multiple_collections_by_id(collection_ids, strict=True):
     """Returns a dict of domain objects representing collections with the
-    given ids as keys. If a collection_id is not present it is not included in
-    the return dict.
+    given ids as keys.
+
+    Args:
+        collection_ids: A list of collection ids of collections to be retrieved
```

Comment:
Add full stops at end of these strings. Ditto below (e.g. in Raises, and in other functions).